server/db/schema.sql

```diff
@@ -1,3 +1,7 @@
+<<<<<<< HEAD
+DROP TABLE IF EXISTS adoption_applications;
+=======
+>>>>>>> 4de76760c73ebdc4e390070a9d06f7d0dd57dabf
```

Comment:
Be careful when handling merge conflicts.  If you don't use the provided buttons you will need to remove lines like those on 1, 3 & 4 yourself.   You will also need to determine the order of the drop table commands.

You will need to start using the `CASCADE` option at the end of certain drop table commands.
https://www.postgresql.org/docs/13/sql-droptable.html